examples/custom-vpc-with-vault/main.tf

```diff
@@ -41,4 +41,6 @@ module "aws_vault" {
   ssh_keys                 = ["ssh-ed25519 AAAAC3Nznte5aaCdi1a1Lzaai/tX6Mc2E+S6g3lrClL09iBZ5cW2OZdSIqomcMko 2 mysshkey"]
   ssh_security_group_ids   = ["${module.aws_vpc.bastion_security_group_id}"]
   vault_ingress_cidr_https = ["0.0.0.0/0"]
+
+  ssl_certificate_id = "arn:aws:acm:eu-central-1:123456789012:certificate/xxxxx-xxxx-xxxx-xxxx-xxxxx"
```

Comment:
I think this is causing the Travis error. Can you make it look valid

Reply:
I haven't checked the failure you mentioned, but we can see that it's now working.